lte/gateway/python/magma/pipelined/tests/test_qfi_gtp.py

```diff
@@ -138,6 +138,7 @@ def test_qfi_delete_tunnel_flows(self):
         snapshot_verifier = SnapshotVerifier(
```

Comment:
This removes the flakiness. But I guess the test does not really test the deletion correctly (if i remove the self.classifier_controller.delete_tunnel_flows line the test is still green).

Reply:
I try to address this issue here https://github.com/magma/magma/pull/13561